includes/plugins/class-jetpack.php

```diff
@@ -113,5 +117,28 @@ public static function fix_instant_search_sidebar_display() {
 		</style>
 		<?php
 	}
+
+	/**
+	 * Disables Google Analytics module. Users will not be able to activate it.
+	 *
+	 * @param array $modules Array with modules slugs.
+	 * @return array
+	 */
+	public static function remove_google_analytics_from_active( $modules ) {
+		return array_diff( $modules, array( 'google-analytics' ) );
+	}
+
+	/**
+	 * Remove Google Analytics from available modules
+	 *
+	 * @param array $modules The array of available modules.
+	 * @return array
+	 */
+	public static function remove_google_analytics_from_available( $modules ) {
+		if ( isset( $modules['google-analytics'] ) ) {
+			unset( $modules['google-analytics'] );
+		}
+		return $module;
```

Comment:
```suggestion
		return $modules;
```

Reply:
wow, thanks for catching that. Dangerous typo!